### PR TITLE
docs: Update README

### PR DESCRIPTION
**Pull request by BenitoVisone**

### CODE REVIEW

File: .github/workflows/code-review.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ on:
     branches:
       - main
       - release-*
+    paths-ignore:
+      - "**.md"
+      - "**.yml"
```
Comment (rodoviti) [inline .github/workflows/code-review.yml]: why ignore only these two type of files and why all yml file in all subpaths?

i suggest to use an "allowed-list" strategy, so:
```suggestion
    paths:
      - "**/src/**"
      - "**pom.xml"
      - ".github/workflows/code-review.yml"
```

File: .github/workflows/build-and-deploy.yml [hunk 2]
```diff
@@ -12,6 +12,9 @@ on:
     branches:
       - "main"
       - "release-*"
+    paths-ignore:
+      - "**.md"
+      - "**.yml"
```
Comment (rodoviti) [inline .github/workflows/build-and-deploy.yml]: why ignore only these two type of files and why all yml file in all subpaths?

i suggest to use an "allowed-list" strategy, so:
```suggestion
    paths:
      - "**/src/main/**"
      - "**pom.xml"
      - "Dockerfile"
      - ".github/workflows/build-and-deploy.yml"
```

**Review by rodoviti — CHANGES_REQUESTED** [review]

File: .github/workflows/build-and-deploy.yml (hunk 2 above)
Comment (BenitoVisone) [reply .github/workflows/build-and-deploy.yml]: fixed with https://github.com/pagopa/pdv-ms-user-registry/pull/19/commits/343cf355b24dcbc9b471f8838ebbdf0522957cf8

File: .github/workflows/code-review.yml (hunk 1 above)
Comment (BenitoVisone) [reply .github/workflows/code-review.yml]: fixed with https://github.com/pagopa/pdv-ms-user-registry/pull/19/commits/343cf355b24dcbc9b471f8838ebbdf0522957cf8

File: README.md [hunk 3]
```diff
@@ -1 +1,76 @@
-# pdv-ms-user-registry
\ No newline at end of file
+# PDV MS-User-Registry
+
+Spring Application implementing the **User-Registry** microservice for the **Personal Data Vault** Project.
+
+This microservice works in conjunction with:
+
+- [pdv-ms-person](https://github.com/pagopa/pdv-ms-person)
+- [pdv-ms-tokenizer](https://github.com/pagopa/pdv-ms-tokenizer)
```
Comment (rodoviti) [inline README.md]: from the point of view of the _tokenizer_, it is not aware of the existence of other microservices in the same _ecosystem_. So, I suggest splitting dependencies into upstream and downstream
```suggestion
---

## Dependencies

Upstream:
- PagoPA products

Downstream:
- [pdv-ms-person](https://github.com/pagopa/pdv-ms-person)
- [pdv-ms-tokenizer](https://github.com/pagopa/pdv-ms-tokenizer)
```

File: README.md [hunk 4]
```diff
@@ -54,23 +54,21 @@ So, first step if you want to test it locally, run these two microservices follo
...
+|-----------------|--------------------------|
+| APP_SERVER_PORT | default: 8080[^app_port] |
+| APP_LOG_LEVEL   | default: DEBUG           |
+
+[^app_port]: When running multiple microservices simultaneously, a different port must be chosen for each one.
 
 From terminal, inside the **app** package:
 
 ```
 ./mvnw spring-boot:run
```
Comment (BenitoVisone) [inline README.md]:
```suggestion
../mvnw spring-boot:run
```

**Review by rodoviti — APPROVED** [review]

**Pull request merged**